react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.getElementById('root');
+const root = ReactDOM.createRoot(container);
+root.render(CustomButton());
```

Comment:
In general, we don't call our own components in React apps. Instead, create a react element

```jsx
root.render(<CustomButton />);
```